app/Models/User.php

```diff
@@ -22,7 +22,9 @@ class User extends Authenticatable
         'email',
         'password',
     ];
-
+    protected $guarded =[
+        'is_admin'
+    ]; // black-list field
```

Comment:
```suggestion
    protected $guarded =[
        'is_admin',
    ]; // black-list field
```
sử dụng cả fillable và guarded trong cùng 1 model => xung đột logic.